theme_beauty/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" inherit_id="website.s_numbers_showcase">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc3" separator=" "/>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Wavy/11','flip':['x']}</attribute>
+    </xpath>
+    <!-- Shape -->
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Wavy_11" style="background-image: url('/web_editor/shape/web_editor%2FWavy%2F11.svg?c1=o-color-1&amp;c4=o-color-4&amp;flip=x'); background-position: 50% 100%;"/>
```

Comment:
- I don't think the `background-position` is necessary ? Since there is no image :slightly_smiling_face: 
- `/web_editor%2FWavy%2F11.svg ` => why is it needed to encode the "/" and "_" this way ? It was done nowhere before :open_mouth: